### ISSUE TITLE
baseURL not set for multihost and multilanguage

### ISSUE THREAD

**Issue by pawelurbanski** [gohugoio/hugo#4378]
I am using Hugo 0.35 64-bit under Windows 10. After my multilingual site with a multihost setup stopped generating content I isolated the following case:
For the configuration as follows:
[languages]
[languages.en]
baseURL = "https://www.pawelurbanski.com"
[languages.pl]
baseURL = "https://www.pawelurbanski.pl"

I get the error that baseURL is not set as a flag nor as a configuration parameter. The debug output shows that Hugo does map languages: en and pl to their respective baseURLs.
The same configuration file used to have more attributes, such as languageName and title, but the result was the same.

The generation does not stop, but proceeds. There are no content files gnerated, but just index files for both languages in their respective subfolders in the public directory. A test static file is copied from the static directory.
I created 4 markdown file with just the basic required frontmatter and content containing just 2 lines of text.
content: index.en.md and index.pl.md are genrated...
/test: test.en.md and test.pl.md are not genreated at all.

The debug output does not show any missing templates. The layout directory has on purpose just _default/single.html and index.html files. There is just a call to: {{ .content }} inside.

I've attached the source directory with my content, config.toml and debug-output.txt file.
[multihost.zip](https://github.com/gohugoio/hugo/files/1692927/multihost.zip)

**Comment by bep** [comment]
I cannot reproduce this with my own site (multilingual). I'm going to assume that there is something odd with your site. 

Could you open a thread on https://discourse.gohugo.io/ and take it from there?

**Comment by pawelurbanski** [comment]
Hi @bep I still have this error. The baseURL is set when I pass it at the commandline. I generated a site using: hugo new site and added the multilanguage block. Hugo throws the same error, while the empty site without a multilanguage block was generated without any issues.
Is there any repository I could clone with your site and check if it generates correctly? I created a thread at the the Discuss form, but still waiting for replies.
